charts/backingservices/README.md

```diff
@@ -39,25 +39,25 @@ Note: Pega does **not** actively update the elasticsearch dependency in `require
-| `srsStorage.provisionInternalESCluster`  | <ul><li>Set to `false` to disable this parameter to use your own Elasticsearch service from the SRS cluster; this Requires you to run `$ make external-es-secrets NAMESPACE=<NAMESPACE_USED_FOR_DEPLOYMENT> PATH_TO_CERTIFICATE=<PATH_TO_CERTS>`. </li><li> PATH_TO_CERTIFICATE takes the location value of the placed </li></ul>   |
-| `domain`                                 | DNS entry associated with externalESDeployment.                                                                                                                                                                                                                                                                                     |
+| Configuration                            | Usage                                                                                                                                                                                                                                                                                                                             |
+|------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `tls`                                    | Setting this to true will make the SRS service use es certificates for connection.                                                                                                                                                                                                                                                |
+| `externalESDeployment.username`          | Give the value of the username as shown in the connection info section of externalESDeployment.                                                                                                                                                                                                                                   |
+| `externalESDeployment.username`          | Give the value of the password as shown in the connection info section of externalESDeployment.                                                                                                                                                                                                                                   |
+| `srsStorage.provisionInternalESCluster`  | <ul><li>Set to `false` to disable this parameter to use your own Elasticsearch service from the SRS cluster; this Requires you to run `$ make external-es-secrets NAMESPACE=<NAMESPACE_USED_FOR_DEPLOYMENT> PATH_TO_CERTIFICATE=<PATH_TO_CERTS>`. </li><li> PATH_TO_CERTIFICATE takes the location value of the placed </li></ul> |
```

Comment:
I'm not 100% sure about this, but do these added details make sense??>> 
`srsStorage.provisionInternalESCluster`  | <ul><li>Set to `false` to disable this parameter and connect to your available Elasticsearch service from the SRS cluster. Disabling this setting requires you to pass an existing TLS certificate in use by your external Elasticseardh service to the SRS cluster using a secret file. To do see, use the command run `$ make external-es-secrets NAMESPACE=<NAMESPACE_USED_FOR_DEPLOYMENT> PATH_TO_CERTIFICATE=<PATH_TO_CERTS>`. </li><li>Here, PATH_TO_CERTIFICATE points to the location value of the placed DNS entry associated with your external Elasticsearch service set using `domain`.</li></ul> |

Reply:
By setting it to `false` the client needs to provide details of the external ES along with the certificates to connect to that elastic search.  The command needs to be run in the given syntax which makes the certs to be available for SRS pods to use for the connection.

Reply:
ok. makes sense. how about this? Get rid of the bullets, since it's all related information, not separate steps or details>>
Set to `false` to disable this parameter and connect to your available Elasticsearch service from the SRS cluster. Disabling this setting requires you to provide connectivity details to your organization's external Elasticsearch service along with an appropriate TLS certificate with which you authenticate with the service. To pass the required certificate to the cluster use a secrets file, run the command:

$ make external-es-secrets NAMESPACE=<NAMESPACE_USED_FOR_DEPLOYMENT> PATH_TO_CERTIFICATE=<PATH_TO_CERTS>

where NAMEPACE is the the deployment namespace running the SRS cluster and PATH_TO_CERTIFICATE points to the location value of the placed DNS entry associated with your external Elasticsearch service set using the `domain` parameter.